python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -149,6 +179,81 @@ async def close_tasks(self):
         await super().close_tasks()
         await self.close_communication()
 
+    async def get_ccw_demand(self):
+        """Get CCW demand.
+
+        The method will get the position of the CCW and of the Rotator and
+        compute an optimum demand for the CCW. It takes into account how far
+        CCW and Rotator are from each other and also how far they both are
+        from the demand position.
+
+        Returns
+        -------
+        ccw_demand_position: `float`
+            CCW demand position (in degrees).
+        ccw_demand_velocity: `float`
+            CCW demand velocity (in degrees).
+
+        """
+
+        rot_data = await self.rotator.tel_Application.next(flush=True)
+        ccw_data = await self.mtmount.tel_Camera_Cable_Wrap.aget()
+
+        if self.last_rotator_position is not None:
+            dt = rot_data.private_sndStamp - self.last_rotator_time
+            # Estimate velocity if GAP between last rotator data and current
+            # data is smaller then MAX_ROTATOR_APPLICATION_GAP, else set
+            # velocity to zero.
+            rotator_velocity = (
+                (rot_data.Position - self.last_rotator_position) / dt
+                if dt < MAX_ROTATOR_APPLICATION_GAP
+                else 0.0
+            )
+        else:
+            rotator_velocity = 0.0
+
+        self.last_rotator_position = rot_data.Position
+        self.last_rotator_time = rot_data.private_sndStamp
+
+        ccw_angle = (ccw_data.CCW_Angle_1 + ccw_data.CCW_Angle_2) / 2.0
```

Comment:
Are you sure about this? I would have expected you might want `ccw_data.CCW_Camera_Position`. If you are unsure I suggest attending the Tuesday morning TMA software meeting to ask, or have Andrew Heyer ask for you. (It doesn't help that there are no Description strings).

My main concern is that I believe CCW has 2 drives and only one is active at a time. It'd be a disaster if only one of these angles was valid depending on which drive was active.

Reply:
I see what you mean.. yes... seems like `CCW_Camera_Position` would be better...